pom.xml

```diff
@@ -164,6 +167,30 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>com.spotify</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>${dockerfile-maven-plugin.version}</version>
+                <configuration>
+                    <imageName>${docker.image.prefix}/${project.artifactId}</imageName>
+                    <dockerDirectory>src/main/docker</dockerDirectory>
```

Comment:
Personally I prefer to store the dockerfile in the root of the app, which is the more widespread convention. Any reason to keep in src/main/docker?

Reply:
There is no specific reason. I have found this convention in several Spring Boot + Docker application. However, I will try to use the root of the app.